### PR TITLE
footer UI fixed

### LINKED ISSUE #408: BUG - footer is not having a good ui/ux
**Describe the bug**
the footer lacks a clean ui and needs more features to make it interactive - social links are also missing as only email is there 

**Expected behavior**
a better footer with cleaner ui and more informative as well
**Screenshots (Mandatory)**

Screenshot 2024-05-13 191744

**Additional Context (Mandatory )**

***Are you contributing under any open-source program ?***
gssoc'24

### PR DESCRIPTION
**Pull request by kairveeehh**
# Title and Issue number 
#408 footer's improvement
Title : Improved the footer

Issue No. : 408

Code Stack : html and css

Close #408 
1. fixed the problem of text line's changing on hovering the text 
2. added social icons for better UI 
3. improved the generalisation of colors
4. added hover effects 
Screenshot 2024-05-15 143748
Screenshot 2024-05-15 143742
Screenshot 2024-05-15 143734# Checklist:

- [X] I have mentioned the issue number in my Pull Request.
- [X] I have commented my code, particularly in hard-to-understand areas
- [X] I have gone through the  `contributing.md` file before contributing

**Additional context (Mandatory )**

***Are you contributing under any Open-source programme?***
GSSOC'24

### CODE REVIEW

**Comment by kairveeehh** [comment]
@apu52  I have recreate a new PR with no conflicts with the main branch and all checks have been passes as well , please make sure to merge it so that no other conflicts arise further 
thankyou , 
kairvee

**Comment by kairveeehh** [comment]
@apu52 please inform

**Comment by kairveeehh** [comment]
> @apu52 please inform and merge

**Comment by apu52** [comment]
@kairveeehh please wait  for sometime .I am reviewing other PRs in this repo .I will also review this PR very soon .Just Don't worry . 

& sorry as you have faced merge conflict problems in the previous PR due to late review of mine.

**Comment by kairveeehh** [comment]
> @kairveeehh please wait  for sometime .I am currently reviewing other PRs in this repo .I will also review this PR very soon .Just Don't worry . 
> 
> & sorry as you have faced merge conflict problems in the previous PR due to late review of mine.

Sure and waiting

File: styles.css [hunk 1]
```diff
@@ -312,7 +312,7 @@ body {
   margin-bottom: 28px;
   background: #FD4766;
   background: linear-gradient(120deg, #1C99FE 20.69%, #7644FF 50.19%, #FD4766 79.69%);;
-  -webkit-background-clip: text;
+
```
Comment (apu52) [inline styles.css:315]: @kairveeehh there is a problem 

image

The Headings are not visible

**Review by apu52 — CHANGES_REQUESTED** [review]
image

in dark theme the other Social icons are not visible , I also want to suggest to change the hover color of social icons according to their custom color (e.g insta--red)

**Comment by kairveeehh** [comment]
> image
> 
> in dark theme the other Social icons are not visible , I also want to suggest to change the hover color of social icons according to their custom color (e.g insta--red)

okay suree , I will look into all these chnages by tonight or morning @apu52

**Comment by kairveeehh** [comment]
@apu52  I am adding tooltips as well , please tell which all social icons you want with the ids and what specific color?
Screenshot 2024-05-30 114625

**Comment by apu52** [comment]
> Author

you can add Insta , facebook , twitter , linkedin as social icons

**Comment by apu52** [comment]
I am closing this PR as this issue has already been exhausted.

**Pull request closed without merging**